provider/sma/device.go

```diff
@@ -0,0 +1,62 @@
+package sma
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/andig/evcc/util"
+	"github.com/imdario/mergo"
+	"gitlab.com/bboehmke/sunny"
+)
+
+// Device holds information for a Device and provides interface to get values
+type Device struct {
+	*sunny.Device
```

Comment:
Können wir dieses "stottern" aus `Device.Device` noch irgendwie auflösen?

Reply:
Muss das `sunny.Device` promoted sein?

Reply:
Dieser Wrapper ist ja nur dafür da um das device threadsafe zu machen bzw unnötig viele Abfragen zum Gerät zu vermeiden. Ich könnte den auch in Wrapper umbenennen aber schön finde ich das dann auch nicht.

Das sunny.Device sollte auch erreichbar sein sonst muss ich zumindest einige Funktionen 1:1 wrappen um sie erreichbar zu machen (siehe meter/sma.go#L136 ff)